src/Core/Types/Types/Descriptors/ArgumentDescriptorBase~1.cs

```diff
@@ -63,6 +63,17 @@ public void Type<TInputType>(TInputType inputType)
             Definition.Type = new SchemaTypeReference(inputType);
         }
 
+        public void Type(ITypeReference typeReference)
```

Comment:
Does this makes sense?

Reply:
Not on public stuff .... the ITypeReference is only for internal purposes ... they basically also are placeholders for the actual type while the schema builder is creating the schema.

Reply:
why do you need it there anyway?